src/main/java/com/google/devtools/build/lib/analysis/actions/CustomCommandLine.java

```diff
@@ -1320,4 +1330,18 @@ public void addToFingerprint(
       }
     }
   }
+
+  /** A {@link PathFragment} that is expanded with {@link PathFragment#getCallablePathString()}. */
+  private static final class CallablePathFragment {
```

Comment:
nit: I got confused for a while because I thought "Callable" means a "implementing the Java Callable interface". But then again, `PathFragment.getCallalbePathString()` is not new in this change, so meh.

Reply:
Shall I change the name?